src/utils/errors/errorHandling.ts

```diff
@@ -178,50 +178,51 @@ export const extractErrorSignature = (message: string): string => {
       return errSig;
     }
   }
-  throw new Error('Unrecognized error signature');
+  return 'Unrecognized error signature';
```

Comment:
should this just return an empty string and let the client decide what to do with it?

Reply:
this is still used internally, we should rework this parts. Now I just tried to be as backward compatible as possible